src/components/molecules/GameCard/GameCard.module.scss

```diff
@@ -2,45 +2,90 @@
 @use 'src/styles/placeholders';
 
 .gameCard {
-    $width: 325px;
-    $height: 400px;
+    $width: 275px;
+    $height: 475px;
     @extend %flex-column;
     gap: 0.5rem;
     padding: 0.5rem;
+    border-radius: 0.5rem;
     width: $width;
     max-width: $width;
     height: $height;
     max-height: $height;
-    color: vars.$text-light;
-    background: vars.$bg-dark;
+    background: vars.$bg-dark-lightest;
 }
 
-.gameCardCoverImg {
-    $height: 200px;
-    width: 100%;
+.gameImgContainer {
+    $height: 250px; // 187px half image height
+    $width: 175px; // 132px half image width
+    width: $width;
+    max-width: $width;
     height: $height;
     max-height: $height;
-    object-fit: fill;
+
+    img {
+        border-radius: 5px;
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        overflow: hidden;
+    }
 }
 
 .gameCardHead {
     @extend %flex-row;
     align-items: center;
     gap: 0.5rem;
+}
 
-    div {
-        @extend %flex-column;
+.headText {
+    @extend %flex-column;
 
-        h2 {
-            $width: 250px;
-            @extend %text-ellipsis;
-            font-size: 1.5rem;
-            width: $width;
-            max-width: $width;
-        }
+    width: calc(
+        100% - 1rem - 50px
+    ); // (ParentWidth - padding&sidebar - Rating component size)
+    h2 {
+        @extend %text-ellipsis;
+        font-size: 1.25rem;
     }
 }
 
 .summary {
     overflow: hidden;
+    width: available;
+    height: 175px;
+}
+
+// Classes & Vars for styles based on rating.
+$high: vars.$color-ufo-green;
+$mid: vars.$color-orange;
+$low: vars.$color-carmine-pink;
+$noRating: #7e8aa9;
+
+$barSize: 0.5rem;
+.sideBarHigh {
```

Comment:
I think the sideBar classes and the Rating classes below are prime real estate for using a Mixin.

Although these are only a 1 property css class, the value comes into play if we ever change/tweak the property itself in the future. Imagine we changed the border to dotted instead of solid (silly, but go with it). If we were using a Mixin you would only have to change the word solid -> dotted once. Every scss class that used the Mixin would automatically get the change.

Reference the Mixin example here https://sass-lang.com/guide#topic-6